config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml

```diff
@@ -45,27 +45,27 @@ patient:
     name_lookup:
     - limit: 1
       search_term: testicular cancer diagnosis 
-  THROUGH_LANES:
-    categories:
-    - biolink:ComplexChemicalMixture
-    - biolink:EnvironmentalExposure
-    enum:
-    type: integer  
-  SPEED:
-    categories:
-    - biolink:ComplexChemicalMixture
-    - biolink:EnvironmentalExposure
-    enum:
-    type: integer 
-  ROADTYPE:
-    categories:
-    - biolink:EnvironmentalExposure
-    enum:
-    - Rural Restricted Access
-    - Rural Unrestricted Access
-    - Urban Restricted Access
-    - Urban Unrestricted Access
-    type: string   

```

Comment:
These three variables exist in covid patient data, so commenting them out in all features yaml file for covid will break the icees-db code when creating a new covid sqlite database. I think we will have to keep them to make the all features yaml file compatible with the current covid data.